keyboards/zen/keymaps/333fred/config.h

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2018 Fredric Silberberg
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the boiler plate with `#pragma once`? 

Eg, remove this: 
```h
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "../../config.h"
```
And the `#endif` at the end with just `#pragma once`